src/tiled/commanddialog.cpp

```diff
@@ -45,6 +45,13 @@ CommandDialog::CommandDialog(QWidget *parent)
 
     setWindowTitle(tr("Edit Commands"));
     Utils::restoreGeometry(this);
+
+    mKeySequenceEdit = mUi->keySequenceEdit;
+
+    connect(mKeySequenceEdit, &QKeySequenceEdit::keySequenceChanged, 
+            mUi->treeView, &CommandTreeView::setShortcut);
```

Comment:
Why does the view have a `setShortcut` method? Please move that to the `CommandDialog`.